api/src/test/resources/org/openmrs/util/databasechange/database2_1To2_20UpgradeTest-dataSet.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
What's the use of this file?

Reply:
I copied this file from the 1_9_0 Upgrade test.

It contains sample data e. .g example orders. I need this because I am updating an order. This order will be set in this file.

Not the complete dataset is necessary but as I said I copied it from the other test.